tools/dataset_converters/semantickitti_converter.py

```diff
@@ -0,0 +1,87 @@
+from os import path as osp
+from pathlib import Path
+
+import mmengine
+
+total_num = {
+    0: 4541,
+    1: 1101,
+    2: 4661,
+    3: 801,
+    4: 271,
+    5: 2761,
+    6: 1101,
+    7: 1101,
+    8: 4071,
+    9: 1591,
+    10: 1201,
+    11: 921,
+    12: 1061,
+    13: 3281,
+    14: 631,
+    15: 1901,
+    16: 1731,
+    17: 491,
+    18: 1801,
+    19: 4981,
+    20: 831,
+    21: 2721,
+}
+fold_split = {
+    'train': [0, 1, 2, 3, 4, 5, 6, 7, 9, 10],
+    'val': [8],
+    'test': [11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21],
+}
+split_list = ['train', 'valid', 'test']
+
+
+def get_semantickitti_info(split):
+    data_infos = dict()
+    data_infos['metainfo'] = dict(DATASET='SemanticKITTI')
+    data_list = []
+    for i_folder in fold_split[split]:
+        for j in range(0, total_num[i_folder]):
```

Comment:
may add a comment to provide an example filename of lidar_points, pts_semantic_mask_path and sample_id